src/cli.ts

```diff
@@ -78,7 +81,9 @@ class Logic {
     try {
       const parsedArgs = Logic.parseArgs();
       output = Merge.merge(Logic.readFiles(parsedArgs.entryDir), parsedArgs.entryObjectName, { skipFUGR: parsedArgs.skipFUGR });
-      output = Merge.appendFooter(output, PackageInfo.version);
+      if (!parsedArgs.noFooter) {
```

Comment:
I'd be more comfortable with "parsedArgs.noFooter === true", as its not a double negation, plus handling of undefined in javascript is always tricky

Reply:
actually `=== false`, well, yea, exactly

Reply:
or well, I cant figure it out

Reply:
Good point. Amended.